### PR TITLE
Design Rework #1 - App icon & texts

### PR DESCRIPTION
**Pull request by LuisMossburger**
**Description**
See also #100 and #118 for progress.
This merges reworked texts with a shared tone that matches our "company values" and the new app launcher icon. Please see the single commits for more information, as more things were done here. In short:
* Proper exceptions for singular/plural actions (e.g. "Regal löschen?" and "Regale löschen?" instead of "Regal(e) löschen?")
* Use a consistent friendly and short tone for all texts that matches our company identity we created on the beginning
* Deleted unused texts, restructured strings.xml
* Updated bookForm to use standard "*" for mandatory fields and moved ISBN field down (we talked about this in team meeting, basically: if I need to use manual adding, not having an ISBN is probably the reason, so this field should not be on top)
* Bugfix for deleting authors (multiple at once not possible) and notes (were not updated after deleting)

Next up: Reworking the help, then colors & UI elements.

**Affects**
Texts in multiple fragments.

**Notes for Reviewer**
Please review & comment/approve - maybe don't be too nitpicky about formulations as I think the changes will need to be discussed in a future team meeting instead of between the two of us here :)

### CODE REVIEW

File: BibBuddy/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <!--<background android:drawable="@color/ic_launcher_background"/-->
-    <foreground android:drawable="@mipmap/ic_launcher_foreground" />
-</adaptive-icon>
+    <background android:drawable="@color/ic_launcher_background"/>
+    <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
+</adaptive-icon>
```
Comment (SarahKurek) [inline BibBuddy/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml:5]:
```suggestion
</adaptive-icon>

```

File: BibBuddy/app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <!-- <background android:drawable="@color/ic_launcher_background"/> -->
-    <foreground android:drawable="@mipmap/ic_launcher_foreground" />
+    <background android:drawable="@color/ic_launcher_background"/>
+    <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
 </adaptive-icon>
```
Comment (SarahKurek) [inline BibBuddy/app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml:5]:
```suggestion
</adaptive-icon>

```

File: BibBuddy/app/src/main/res/values/ic_launcher_background.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="ic_launcher_background">#FFFFFF</color>
+</resources>
```
Comment (SarahKurek) [inline BibBuddy/app/src/main/res/values/ic_launcher_background.xml:4]:
```suggestion
</resources>

```

File: BibBuddy/app/src/main/res/layout/fragment_author_form.xml [hunk 4]
```diff
@@ -34,12 +34,19 @@
                 >
 
                 <!-- title -->
+                <TextView
+                        android:text="@string/book_form_notice"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:id="@+id/textView2"
```
Comment (SarahKurek) [inline BibBuddy/app/src/main/res/layout/fragment_author_form.xml:41]:
```suggestion
                        android:id="@+id/info_mandatory"
```

**Review by SarahKurek — COMMENTED** [review]
Thanks for fixing the bug with deleting the authors and cleaning up the strings file. Please add a new line at the end of the xml files and possibly rename the one commented id, everything else fits.

**Comment by LuisMossburger** [comment]
THanks for the review! Adjusted everything and additionally also changed the ID of the "mandatory" text in the book form.

File: BibBuddy/app/src/main/res/layout/fragment_author_form.xml (hunk 4 above)
Comment (LuisMossburger) [reply BibBuddy/app/src/main/res/layout/fragment_author_form.xml]: Did adjust & also added an ID in the book form xml.

File: BibBuddy/app/src/main/res/values/ic_launcher_background.xml (hunk 3 above)
Comment (LuisMossburger) [reply BibBuddy/app/src/main/res/values/ic_launcher_background.xml]: Check!

File: BibBuddy/app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml (hunk 2 above)
Comment (LuisMossburger) [reply BibBuddy/app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml]: Check!

File: BibBuddy/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml (hunk 1 above)
Comment (LuisMossburger) [reply BibBuddy/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: Check!

**Review by SarahKurek — APPROVED** [review]
Now it fits.

**Pull request merged**